src/Esfa.Vacancy.Register.Api/Web.config

```diff
@@ -17,7 +17,7 @@
     <add key="EnvironmentName" value="DEV" />
     <add key="LoggingRedisKey" value="logstash" />
     <add key="LoggingRedisConnectionString" value="" />
-    <add key="AVMSPLUSCONNECTIONSTRING" value="" />    
+    <add key="DasApprenticeshipInfoApiBaseUrl" value="http://das-pp-apprenticeshipinfoservice.cloudapp.net/" />
```

Comment:
This is in production so you can remove the setting and the no arg constructor will use prod